tests/integration/unbonding.go

```diff
@@ -461,3 +461,47 @@ func (s *CCVTestSuite) TestRedelegationProviderFirst() {
 	// Check that ccv unbonding op has been deleted
 	checkCCVUnbondingOp(s, s.providerCtx(), s.consumerChain.ChainID, valsetUpdateID, false)
 }
+
+func (s *CCVTestSuite) TestTooManyLastValidators() {
```

Comment:
Might make sense to have consistent names with this [E2E test](https://github.com/cosmos/interchain-security/pull/1942/files). Either name is fine by me. (cc @p-offtermatt )

Reply:
I think it is probably ok to have different names, since we don't have common naming conventions across e2e and integration tests right now, anyways